lib/AddUserCmd.pm

```diff
@@ -0,0 +1,30 @@
+package AddUserCmd;
+
+use strict;
+use warnings;
+use parent 'Command';
+
+use UserService;
+use Output;
+use Error;
+
+our @ISA = qw(Command);
+
+
+sub excecute {
```

Comment:
@manuellerena I just see this, but the function name must be 'execute' not 'excecute' ;)